app/src/main/java/com/technopark/hw_1/DataSource.java

```diff
@@ -0,0 +1,34 @@
+package com.technopark.hw_1;
+
+import android.graphics.Color;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataSource {
+
+    private final List<NumberModel> mData;
+
+    public DataSource(int size) {
+        mData = new ArrayList<>();
+
+        for (int i = 1; i <= size; i++) {
+            NumberModel model = new NumberModel(i, i % 2 == 0 ? Color.RED : Color.BLUE);
+            mData.add(model);
+        }
+    }
+
+    public void AddNextNumber() {
```

Comment:
let's keep method names in the same style. e.g. in `camelCase` with lower-cased first letter